app/assets/javascripts/angular/services/graph_refresher.js

```diff
@@ -21,9 +21,10 @@ angular.module("Prometheus.services").factory('GraphRefresher',
         cache: false
       }).then(function(payload, status) {
         data = payload.data;
+        var errMsg;
```

Comment:
Same comment applies here. Actually, JavaScript's scoping is such that a variable defined in the `switch` below will be visible also outside of the `switch`. Same for loops. Probably that's why. Meh.